lib/icinga/compatutility.cpp

```diff
@@ -300,12 +300,12 @@ int CompatUtility::GetCheckableIsVolatile(const Checkable::Ptr& checkable)
 
 double CompatUtility::GetCheckableLowFlapThreshold(const Checkable::Ptr& checkable)
```

Comment:
Can we please remove those CompatUtility::* methods? :)

Reply:
This is a different topic